services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,28 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
+  xQueueSend(thermalMgrQueueHandle, event, portMAX_DELAY);
 
   return ERR_CODE_SUCCESS;
 }
 
 void osHandlerLM75BD(void) {
-  /* Implement this function */
+  float temp;
+  readTempLM75BD(LM75BD_OBC_I2C_ADDR, &temp);
```

Comment:
You should avoid reading from the temp sensor in an ISR for 2 reasons

1) usually I2c and other serial communication functions will be protected with a mutex and we should not be trying to take a mutex in an ISR since it is running outside of a task

2) I2C and consequently this function call are relatively slow and the ISR should be kept as short and fast as possible

It would be better to notify the thermal mgr task somehow that this interrupt has happened so that it can deal with it outside of an ISR context